moz_kinto_publisher/test_publisher.py

```diff
@@ -45,17 +48,21 @@ def make_record(run_id, *, parent, channel):
 
 
 class MockRunDB(main.PublishedRunDB):
-    def __init__(self, runs):
-        self.run_identifiers = runs
-
-    def get_run_timestamp(self, run_id):
-        return timestamp_from_run_id(run_id)
-
-    def is_run_valid(self, run_id, channel):
-        return run_id in self.run_identifiers
-
-    def is_run_ready(self, run_id):
-        return True
+    def __init__(self, runs, completed_runs, channel=main.CHANNEL_ALL):
+        for run in runs:
+            os.mkdir(Path(run))
```

Comment:
Do we need to register any kind of cleanup function for this?

Reply:
Nope, we get a fresh temporary directory for each test.